### PR TITLE
Tommy/backend_pagination_artists

### PR DESCRIPTION
**Pull request by tommypoa**
### Related PRs
None.
### Migrations
None.
### Tests Performed, Edge Cases
Tested as different users – admin, artist and buyer.

CC: @by-co @aivantg

### CODE REVIEW

**Comment by by-co** [comment]
@tommypoa Mind resolving these merge conflicts? I think they're from #124 and #115 :)

**Comment by by-co** [comment]
Actually I would recommend rebasing on `master` — for some reason it's still showing the `Gemfile` changes eve though #115 is merged already 🤔

File: app/controllers/api/artists_controller.rb [hunk 1]
```diff
@@ -15,15 +15,30 @@ def show
   end
 
   def index
-    artists = Artist.all
-    render json: artists
+    artists = current_admin ?
```
Comment (by-co) [inline app/controllers/api/artists_controller.rb]: Woah, love this Rails-side admin check 🙌

**Comment by tommypoa** [comment]
Merge conflicts fixed!

**Pull request merged**